### PR TITLE
Feature/samtools sort

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### This PR fixes:

- Replaced sambamba sort with samtools sort after alignment
- Replaced recipe picartools_mergesamfiles with samtools_merge

### How to test:

- Automatic and continuous test pass

### Expected outcome:
- Installation, unit and integration tests pass

### Review:
- [x] Code review
- [x] New code is executed and covered by tests
- [x] Tests pass

### CODE REVIEW

File: lib/MIP/Recipes/Analysis/Bwa_mem.pm [hunk 1]
```diff
@@ -324,7 +325,7 @@ sub analysis_bwa_mem {
 
         ## Prepare for downstream processing
         # Can be either infile or instream
-        my $sambamba_sort_infile;
+        my $samtools_sort_infile;
```
Comment (jemten) [inline lib/MIP/Recipes/Analysis/Bwa_mem.pm]: Isn't it better to declare the variable at the same time as you define it at line 361?

File: lib/MIP/Recipes/Analysis/Bwa_mem.pm [hunk 2]
```diff
@@ -357,21 +358,23 @@ sub analysis_bwa_mem {
         print {$filehandle} $PIPE . $SPACE;
 
         ## Set sambamba sort input; Pipe from samtools view
```
Comment (jemten) [inline lib/MIP/Recipes/Analysis/Bwa_mem.pm]:
```suggestion
        ## Set samtools sort input; Pipe from samtools view
```

File: lib/MIP/Recipes/Analysis/Bwa_mem.pm [hunk 3]
```diff
@@ -798,20 +803,22 @@ sub analysis_run_bwa_mem {
         say   {$filehandle} $NEWLINE;
 
         ## Set sambamba sort input; Sort directly from run-bwakit
```
Comment (jemten) [inline lib/MIP/Recipes/Analysis/Bwa_mem.pm]:
```suggestion
        ## Set samtools sort input; Sort directly from run-bwakit
```

File: lib/MIP/Recipes/Analysis/Bwa_mem.pm [hunk 4]
```diff
@@ -798,20 +803,22 @@ sub analysis_run_bwa_mem {
         say   {$filehandle} $NEWLINE;
```
Comment (jemten) [inline lib/MIP/Recipes/Analysis/Bwa_mem.pm]:
```suggestion
        say {$filehandle} $NEWLINE;
```

File: lib/MIP/Recipes/Analysis/Bwa_mem.pm [hunk 5]
```diff
@@ -778,7 +783,7 @@ sub analysis_run_bwa_mem {
 
         ## Prepare for downstream processing
         # Can be either infile or instream
-        my $sambamba_sort_infile;
+        my $samtools_sort_infile;
```
Comment (jemten) [inline lib/MIP/Recipes/Analysis/Bwa_mem.pm]: same comment as above

File: lib/MIP/Recipes/Analysis/Bwa_mem.pm (hunk 1 above)
Comment (henrikstranneheim) [reply lib/MIP/Recipes/Analysis/Bwa_mem.pm]: 👍

Comment (henrikstranneheim) [reply lib/MIP/Recipes/Analysis/Bwa_mem.pm]: I will remove. Think it was a remnant of when the two recipes were one

File: lib/MIP/Recipes/Analysis/Bwa_mem.pm (hunk 4 above)
Comment (henrikstranneheim) [reply lib/MIP/Recipes/Analysis/Bwa_mem.pm]: This is perltidy trying to align "{".

File: lib/MIP/Recipes/Analysis/Bwa_mem.pm (hunk 2 above)
Comment (henrikstranneheim) [reply lib/MIP/Recipes/Analysis/Bwa_mem.pm]: Done

File: lib/MIP/Recipes/Analysis/Bwa_mem.pm (hunk 3 above)
Comment (henrikstranneheim) [reply lib/MIP/Recipes/Analysis/Bwa_mem.pm]: Done

File: lib/MIP/Recipes/Analysis/Bwa_mem.pm (hunk 5 above)
Comment (henrikstranneheim) [reply lib/MIP/Recipes/Analysis/Bwa_mem.pm]: Done

**Pull request merged**